### DISCUSSION TITLE
Git error - never seen before

### DISCUSSION THREAD

**Discussion by The-Futurist** [community/community#21718] [New to GitHub]
<p>I’m in SmartGit and doing a pull on a repo, it fails with:</p>
<p><em>git couldn’t find remote ref refs/heads/project_one</em></p>
<p>However the branch “project_one” has been deleted from the main GitHub repo and from my GitHub fork and locally from my local repo and from the remotes for the main repo and the fork, there is no trace of the branch name anywhere so far as I can tell.</p>
<p>I even scanned the .git folder searching all files for the text “project_one” and found nothing.</p>
<p>I’m recloning the repo but I really would like to know why I am seeing this…</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>If you can’t reproduce the problem using git from the command-line, then I would think that it is a bug in SmartGit. You may want to report it to them.</p>
<p>Sorry I don’t have more information to give.</p>
